packages/office-ui-fabric-react/src/components/Pivot/Pivot.scss

```diff
@@ -82,6 +82,11 @@
   &.is-disabled {
     color: $ms-color-neutralTertiary;
   }
+  
+  // Spacing between text and count 
+  .ms-Pivot-count {
+    @include margin-left(4px);
```

Comment:
Does the text change order on RTL? E.g. "(4) New items"

If not, this can simply be `margin-left: 4px` without a need for the RTL mixin.